### DISCUSSION TITLE
useSortBy set the preferred index

### DISCUSSION THREAD

**Discussion by JSShant** [algolia/instantsearch#5812] [Q&A] [answered]
I have a webpage which has either items or sales, which are switched through a tab.
Using useSortBy with my current dropdown i have been able to filter items on either page by creating replica indexes.
There is an index for price high to low, low to high, closing soon etc.
When on the items page the initial sort is using the main index which is intended, however when switching to the sales page i would like the index used to population the page to be one of the replica indexes not the main one.
 const {  currentRefinement } = useSortBy();
 Currently currentRefinement is always pointing towards the main index, how can i change this behaviour only for the sales page.

**Comment by Haroenv** [comment, marked as answer]
The default index is the one you pass to `InstantSearch`, so you can change that depending on which page you're showing
